routes/web.php

```diff
@@ -21,8 +21,8 @@
 // });
 
 Route::get('/', function () {
-    return view('welcome');
-});
+    return view('dashboard');
+})->middleware(['auth']);
```

Comment:
With the addition of the register page onto the login page, this works, however, is optional.